plop-load.md

```diff
@@ -1,76 +1,97 @@
 
 ### plop.load([targets](#targets), [[config](#config)], [[include](#include)])
 
 #### targets
+
 - **type:** `String` or `Array<String>`
 - **required**
 
 `targets` is the location or locations of the plopfile(s) to be loaded. These locations can be file paths (absolute or relative to the current plopfile) or the node module name of the `plop-pack` that should be loaded.
 
 #### config
+
 - **type:** `Object`
 - **optional**
 
 `config` is an object that can be passed to the plopfile or `plop-pack` when they are run. This allows the consumer of the plopfile or `plop-pack` to configure certain aspects of its functionality. To know what properties should be in this object, see the documentation provided by the author.
 
 #### include
+
 - **type:** `Object`
 - **default:** `{ generators:true, helpers:false, partials:false, actionTypes:false }`
 - **optional**
 
 `include` is an object that can contain 4 properties (`generators`, `helpers`, `partials`, and `actionTypes`). Each of these properties should have an [`IncludeDefinition`](#IncludeDefinition) as its value. Most of the time this object is not needed because the plopfile or `plop-pack` is able to specify a default [`IncludeDefinition`](#IncludeDefinition) to be used.
 
 #### Interface `IncludeDefinition`
+
 - **Boolean:** `true` will include all assets, `false` will include non of them.
 - **Array:** a list of asset names that should be included. any assets that don't match a name in this list will be skipped.
 - **Object:** the include object allows the consumer to rename assets when for use in their own plopfile. the property name in this object is the asset name, the value is the name that will be given to the asset when loaded.
 
 ## Examples
-*load via a path*
+
+_load via a path_
+
 ```javascript
-	// loads all 5 generators, no helpers, actionTypes or partials (even if they exist)
-	plop.load('./plopfiles/component.js');
+// loads all 5 generators, no helpers, actionTypes or partials (even if they exist)
+plop.load('./plopfiles/component.js');
 ```
-*load via a path with a custom include config*
+
+_load via a path with a custom include config_
+
 ```javascript
-	// loads all helpers, no generators, actionTypes or partials (even if they exist)
-	plop.load('./plopfiles/component.js', {}, { helpers: true });
+// loads all helpers, no generators, actionTypes or partials (even if they exist)
+plop.load('./plopfiles/component.js', {}, {helpers: true});
 ```
-*load via a path with a limited include*
+
+_load via a path with a limited include_
+
 ```javascript
-	// loads only the "js-header" helper, no generators, actionTypes or partials (even if they exist)
-	plop.load('./plopfiles/component.js', {}, { helpers: ['js-header'] });
+// loads only the "js-header" helper, no generators, actionTypes or partials (even if they exist)
+plop.load('./plopfiles/component.js', {}, {helpers: ['js-header']});
 ```
-*load via a path with config object*
+
+_load via a path with config object_
```

Comment:
use `_` character for italic as `*` almost everywhere is used for display text as bold.
ref: https://guides.github.com/features/mastering-markdown/#Emphasis

Reply:
MD supports either or. I'll allow this change to stay, but in the future I'd suggest avoiding bikeshedding this

Reply:
this is done by prettier also